File: src/main/java/com/p2p/lending/service/BorrowmoneyService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Approveitem;
import com.p2p.lending.pojo.Borrowmoney;
import com.p2p.lending.pojo.Log;

import com.github.pagehelper.PageInfo;

public interface BorrowmoneyService {

	public List<Borrowmoney> findList(Map<String, Object> map);

	PageInfo<Borrowmoney> findList(Borrowmoney Borrowmoney, Integer pageNo, Integer pageSize);

	int delete(Integer id);

	int create(Borrowmoney record);

	Borrowmoney get(Integer id);

	int update(Borrowmoney record);

	/*
	 * ygx 2017 2 25 效仿陈庆山分页
	 */
	public List<Borrowmoney> pagingSel(Map<String, Object> map);
	/*
	 * zq 2017年3月3日10:40:48 
	 */
	public void toaddborr(Borrowmoney borrowmoney);
	public List<Borrowmoney> updhuankuan();
	public Borrowmoney borrowget(Integer ids);

	
	/**
	 * hjy
	 */
	Map<String, Object> selecthjy(String currpage);
}

File: src/main/java/com/p2p/lending/service/BorrowmoneyService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Approveitem
import com.p2p.lending.entity.Borrowmoney
import com.p2p.lending.entity.Log

import com.github.pagehelper.PageInfo

interface BorrowmoneyService {

    fun findList(map: Map<String, Any>): List<Borrowmoney>

    fun findList(Borrowmoney: Borrowmoney, pageNo: Int?, pageSize: Int?): PageInfo<Borrowmoney>

    fun delete(id: Int?): Int

    fun create(record: Borrowmoney): Int

    operator fun get(id: Int?): Borrowmoney

    fun update(record: Borrowmoney): Int

    /*
	 * ygx 2017 2 25 效仿陈庆山分页
	 */
    fun pagingSel(map: Map<String, Any>): List<Borrowmoney>

    /*
	 * zq 2017年3月3日10:40:48
	 */
    fun toaddborr(borrowmoney: Borrowmoney)

    fun updhuankuan(): List<Borrowmoney>
    fun borrowget(ids: Int?): Borrowmoney


    /**
     * hjy
     */
    fun selecthjy(currpage: String): Map<String, Any>
}